index.html

```diff
@@ -6,7 +6,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
-    <title>Microverse First Project</title>
+    <title>Personal portfolio: porfolio to show case working experience</title>
```

Comment:
- [ ] It would be a good thing to personalize the title of your website portfolio. A good example is "Ridwan's Personal Portfolio".